src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/pullrequest/BitbucketServerPullRequest.java

```diff
@@ -113,6 +115,14 @@ public void setAuthor(Author author) {
             authorLogin = null;
         }
     }
+    
+    public boolean isCanMerge() {
```

Comment:
This should be marked as `@CheckForNull` as it is known to be null if the "Call Can Merge" option is disabled.

Reply:
a boolean cannot be null, true or false :)

Reply:
Heh, right :) Then it should be a `Boolean`